concrete_datastore/api/v1/authentication.py

```diff
@@ -95,3 +95,16 @@ def authenticate_credentials(self, key):
             expire_secure_token(secure_token)
 
         return (token.user, token)
+
+
+class URLTokenExpiryAuthentication(TokenExpiryAuthentication):
```

Comment:
Please explain within a comment the purpose of that class

Reply:
docstring added